pkg/storaged/dialog.jsx

```diff
@@ -1107,27 +1107,27 @@ const UsersPopover = ({ users }) => {
             bodyContent={
                 <>
                     { services.length > 0
-                        ? <p>
-                            <b>{_("Services using the location")}</b>
+                        ? <>
+                            <p><b>{_("Services using the location")}</b></p>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231025-042831-94ae7a2d-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)